analyses/proteomic-data/proteomicsGSVA.R

```diff
@@ -0,0 +1,108 @@
+
+#load libraries
+library(tidyverse)
+library(readr)
+library(tibble)
+library(msigdbr)
+library(GSVA)
+
+
+#Magrittr pipe
+`%>%` <- dplyr::`%>%`
+#
+
+
+#set directories
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+#data directory
+data_dir <- paste(root_dir, "/data", sep="")
+#directory containing input files
+inputfile_dir <- "./output"
+#directory to store output
+output_dir <- "./GSEAoutput"
```

Comment:
Probably we can define the directories using file.path, instead of using paste

for example:
data_dir <- file.path(root_dir, "data")

So it will work on different operation systems

Reply:
Changed this to use file.path as you suggested.